packages/mdc-textfield/README.md

```diff
@@ -340,6 +366,11 @@ complicated.
 | removeClass(className: string) => void | Removes a class from the root element |
 | addClassToLabel(className: string) => void | Adds a class to the label element. We recommend you add a conditional check here, and in `removeClassFromLabel` for whether or not the label is present so that the JS component could be used with text fields that don't require a label, such as the full-width text field. |
 | removeClassFromLabel(className: string) => void | Removes a class from the label element |
+| eventTargetHasClass(target: HTMLElement, className: string) => boolean | Returns true if classname exists for a given target element |
+| registerTextFieldInteractionHandler(handler: EventListener) => void | Registers an event handler on the root element for a "click" event |
```

Comment:
Why are we calling this `...InteractionHandler` if it is specifically for `click` events? This is inconsistent with our own recommended practices and usage of this term in many other components, which pertains to a function which takes both an event type and handler as input.

Compare to our own usage of the term in:

* authoring-components.md
* Vue framework examples
* Many existing components, including mdc-dialog, mdc-drawer, mdc-form-field, mdc-icon-toggle, mdc-menu, mdc-ripple, mdc-select, mdc-slider, mdc-snackbar, mdc-tabs

Reply:
Yeah, you're right.